server/network/aoprotocol.py

```diff
@@ -452,6 +452,9 @@ def net_cmd_ms(self, args):
                     "While that is not a blankpost, it is still pretty spammy. Try forming sentences."
                 )
                 return
+        if text.lstrip().startswith('(('):
+            self.client.send_ooc("Please, *please* use the OOC chat instead of polluting IC. Normal OOC is local to area. You can use /g to talk across the entire server.")
+            return
```

Comment:
i feel like this might get us some backlash for messing up people's ways of doing things - can we make it warn the user only once and then never again for the remainder of the session

Reply:
d'aww fine I'll make it a separate PR
~~you're no fun~~